tests/test_bootstrap.py

```diff
@@ -598,7 +598,7 @@ def mock_is_virtual_env() -> Generator[Mock]:
 
 
 @pytest.fixture
-def mock_enable_logging() -> Generator[Mock]:
+def mock_enable_logging() -> Generator[AsyncMock]:
```

Comment:
Does it make sense to update all the type hints in the file?
`mock_enable_logging: AsyncMock`

Or do we want to do that later to avoid merge conflict?

Reply:
Let's do it here too, I just forgot